README.md

```diff
@@ -10,6 +10,53 @@ Install via npm:
 $ npm i @godaddy/split-data-loader --save
 ```
 
+## Usage
+
+Use this package if you are using Split.io for experimentation on a webpage and want to take advantage of browser caching with [`localStorage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage).
+
+Please note that this package is intended to be used in tandem with a split data serializer to create the `serializedData` parameter properly.
+An example of such serializer is [`split-node-serializer`](https://github.com/godaddy/split-node-serializer).
+
+```js
+import loadDataIntoLocalStorage from '@godaddy/split-data-loader'
+
+loadDataIntoLocalStorage({
+  serializedData: {
+    segmentsData: {
+      "segment_1": ['test-visitor-1', 'test-visitor-2', 'test-shopper-1'],
```

Comment:
this will actually look like:
`"segment_1": "{ name: 'segment_1', added: ['test-visitor-1', 'test-visitor-2', 'test-shopper-1']}"`